jwtauth.go

```diff
@@ -68,53 +98,33 @@ func NewWithParser(alg string, parser *jwt.Parser, signKey []byte, verifyKey []b
 // http response.
 func Verifier(ja *JwtAuth) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
-		return Verify(ja, "")(next)
+		return Verify(ja, ExtractQuery, ExtractHeader, ExtractCookie)(next)
 	}
 }
 
-func Verify(ja *JwtAuth, paramAliases ...string) func(http.Handler) http.Handler {
+func Verify(ja *JwtAuth, extractors ...ExtractorFunc) func(http.Handler) http.Handler {
```

Comment:
And just use `func(r *http.Request) string` as parameter, which isn't much larger or complicated:
`func Verify(ja *JwtAuth, tokenSources ...func(r *http.Request) string) func(http.Handler) http.Handler {...}`